### ISSUE TITLE
[Bug]: Chinese translation was chosen, but the status of shrubs and trees was not translated into Chinese

### ISSUE THREAD

**Issue by 186-hungry-tiger** [Annosz/UIInfoSuite2#515]
### Mod Version

2.3.2

### Multiplayer

Single Player

### Description

Chinese translation was chosen, but the status of shrubs and trees was not translated into Chinese

**Comment by drewhoener** [comment]
I don't believe anyone has updated these translation entries yet, I encourage you to open a PR if you speak the language!

**Comment by 186-hungry-tiger** [comment]
If you have any plans to translate these, I am willing to provide you with assistance in translation work, but I do not know programming

**Comment by 186-hungry-tiger** [comment]
> 我不相信有人更新了这些翻译条目，如果您会说这种语言，我鼓励您开设 PR！

If you have any plans to translate these, I am willing to provide you with assistance in translation work, but I do not know programming

**Comment by 53ndC0r30n1y** [comment]
[Go to code](https://github.com/Annosz/UIInfoSuite2/blob/8c650efd003bb01b5fdb3f7bf453d4b0060fbf8f/UIInfoSuite2/UIElements/ShowCropAndBarrelTime.cs#L484)
```
  private static string GetTreeTypeName(string treeType)
  {
    switch (treeType)
    {
      case "1": return "Oak";
      case "2": return "Maple";
      case "3": return "Pine";
      case "6": return "Palm";
      case "7": return "Mushroom";
      case "8": return "Mahogany";
      case "9": return "Palm (Jungle)";
      case "10": return "Green Rain Type 1";
      case "11": return "Green Rain Type 2";
      case "12": return "Green Rain Type 3";
      case "13": return "Mystic";
      default: return $"Unknown (#{treeType})";
    }
  }
```
Looks like it can't be translated.
看起啦没法直接翻译

**Comment by 186-hungry-tiger** [comment]
I'm sorry I only saw it now

Oak-橡树
Maple-枫树
Pine-松树
Palm-棕榈树
Mushroom-蘑菇
Mahogany-桃花心木
Palm (Jungle)-丛林棕榈树
Green Rain Type 1-苔雨树（类型1）
Green Rain Type 2-苔雨树（类型2）
Green Rain Type 3-苔雨树（类型3）
Mystic-神秘树
